en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,322 @@
+
+Cut the USB cable of the camera so that the length is 15 cm. Cut the USB-Adapter cable to be 25 cm and solder the two cables together.
+To install the camera you need to cut a hole into the foam of the wall.
+Then you can mount the camera with double sided tape to the wall.
+
+Hole to feed the USB cable through the wall.
+
+![USB Camera 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/camera-01.jpg)
+
+### Flight Controller
+
+The Flight controller can be installed above the ESC.
+If a Pixhawk 6c or 6c mini is used, simply stick the flight controller with double sided tape into place.
+
+If a Skynode is used, place it at the on top of the ESCs and mark the 2 rear mounting locations on the injection molded plastic part of the ZMO.
+Remove the Skynode from the vehicle and drill 2 holes with a 2.8 mm drill bit into the plastic part.
+Put the Skynode back into place and screw it down with 2x M3x10 screws.
+An other option is to add some threaded inserts into the holes. Since the injection molded part of the ZMO is very thin, they need to be glued in place. Screw the front Skynode mount with 2x M3x10 screws at the Skynode.
+Then add some 5 min epoxy at the bottom of the mount and put a weight on top of the Skynode until the glue is cured.
+To better reach the 2 mounting screws at the front, poke 2 holes from the top through the foam so you can access the screws.
+
+Mounting holes for the Skynode in the back
+
+![Flight controller 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-01.jpg)
+
+Skynode mount in the front
+
+![Flight controller 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-02.jpg)
+
+### Antennas and RC Receiver
+
+One LTE antenna can be installed at the bottom of the vehicle. For that you can feed the antenna wire through the opening for the ESC heat-sink.
+The second antenna can be installed on the inside of the vehicle on the left side of the battery compartment.
+The RC receiver can also be placed at the left side of the battery compartment.
+
+LTE antenna 1
+
+![LTE antenna 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-01.jpg)
+
+LTE antenna 2 and RC receiver
+
+![LTE antenna 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-02.jpg)
+
+## Software Setup
+
+### Select Airframe
+
+1. Open QGC and go under vehicle setup.
+1. Search for [SYS_AUTO](../advanced_config/parameter_reference.md#SYS_AUTOSTART) in the parameters tab.
+1. Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `13300`.
+1. Reboot the vehicle.
+
+![SYS_AUTO](../../assets/airframes/vtol/omp_hobby_zmo_fpv/sys_auto.png)
+
+### Sensor Calibration
+
+First make sure to set the [correct orientation of the flight controller](../config/flight_controller_orientation.md).
+This should be the default (`ROTATION_NONE`).
+
+Then calibrate the main sensors:

```

Comment:
FYI I modified this a bit to explain in more detail what you do - more importantly to point to the relevant docs. NOTE, I am assuming mounting is as ROTATION_NONE